app/views/locations/show.html.erb

```diff
@@ -52,16 +52,13 @@
 
     <p>The appointment times available can change depending on your location.</p>
 
-    <% if @location.booking_location %>
-      <p>
-        Tell the operator that you want to book a Pension Wise appointment at
-        <%= @location.name %> Citizens Advice.
-      </p>
-    <% end %>
+    <h2>Before you book</h2>
+
+    <p>You should be aged <b>50 or over</b> and have a <a href="/en/pension-types">defined contribution</a> pension (not a final salary or career average pension).</p>
```

Comment:
This ought to use the path helper so when this is translated everything Just Works ™️